Create UltraPunishments_Brazil.lang

ReEvadere opened the pull request:
Added Brazil Lang for UltraPunishments. Translations created by Discord User: tangiershk

DAmNRelentless reviewed (changes requested):
Please remove the spaces between the styling characters.
It has to look like this `**example**` and not like this `** example **`.
Additionally, you have to remove the `#` symbol in front of each line.

Serviettesky commented:
Done

DAmNRelentless reviewed (approved):
Thanks for your contribution!

ReEvadere committed:
Create UltraPunishments_Brazil.lang

Added Brazil Lang for UltraPunishments. Translations created by Discord User: tangiershk
